.github/ISSUE_TEMPLATE/bug_issue_template.yml

```diff
@@ -48,6 +54,16 @@ body:
     render: markdown
   validations:
     required: false
+- type: textarea
+  attributes:
+    label: Criteria for Completion
+    description: This should be a bulleted list of items that, if satisfied, would result in an acceptably complete Pull Request. Be as specific as you need to be.
+    placeholder: |
+      - [ ] When a user clicks on {this} it should now do {that}
+      - [ ] This feature should only be available to organization admins
+      - [ ] Add a test proving that it works
+  validations:
+    required: false
```

Comment:
@meg-gutshall could you add this also in the Feature Request one too?

Reply:
It is in there. Has been the whole time. Line 34

Reply:
Ahh thanks! I missed it sorry bout that.